package.json

```diff
@@ -60,7 +61,7 @@
     "typescript": "^4.8.4"
   },
   "peerDependencies": {
-    "@capacitor/core": "^3.0.0 || ^4.0.0"
+    "@capacitor/core": "5.0.0-beta.2" 
```

Comment:
This breaks when trying to use "next" version - [5.0.0-rc.1](https://github.com/ionic-team/capacitor/releases/tag/5.0.0-rc.1)

Reply:
Why do we need to enforce capacitor v5 as a peer dependency? My assumption is that the plugin is compatible with v3+ and we should avoid peer dependency warning when the plugin installed with the capacitor v3-4

Reply:
Tested this plugin PR with the capacitor 4.x and all looks good.